libs/output-mapping/tests/Writer/StorageApiLocalTableWriterTest.php

```diff
@@ -884,7 +884,10 @@ public function testWriteTableOutputMappingWithEmptyStringPkInManifest(): void
         $jobIds = $tableQueue->waitForAll();
         self::assertCount(1, $jobIds);
 
-        $writer = new TableWriter($this->getLocalStagingFactory(null, 'json', $testLogger));
+        $root = $this->createTemp()->getTmpFolder();
+        file_put_contents($root . '/upload/table11.csv', "\"Id\",\"Name\"\n\"test\",\"test\"\n");
+
```

Comment:
Kvůli sliceru musí být data v jiném hrníčku. Jinak to failuje. Slicer přepisuje data/manifest v původním umístění.